creation/web_base/utils_gs_signals.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+#*******************************************************************#
+# utils_gs_signals.sh                                               #
+# This script contains signals' utility functions for the           #
+# glidein_startup.sh script                                         #
+#*******************************************************************#
+
+################################
+# Forwards signals to the children processes
+# Arguments:
+#   1: signal
+# Globals (r/w):
+#   ON_DIE
+# Used:
+#   GWMS_MULTIGLIDEIN_CHILDS
```

Comment:
GWMS_MULTIGLIDEIN_CHILDS  -> SIGNAL_CHILDREN_LIST
This needs to happen also in glidein_startup.sh
There could be also a `signal_add_child()` and `signal_set_children()`